static/js/settings/languages.js

```diff
@@ -0,0 +1,52 @@
+/*global each, category_template, $categories_form */
+window.addEventListener("DOMContentLoaded", function(){
+    language_template = document.querySelector("template#language_template").innerHTML;
+    $categories_form = $("form#languages tbody");
+});
+
+function add_language(event){
+    event.preventDefault();
+    var $new_language = $(language_template);
+    $categories_form.append($new_language);
+    $new_language.find("[data-toggle=\"tooltip\"]").tooltip();
+}
+
+function delete_language(event, button){
```

Comment:
Codacy found an issue: ['delete_language' is defined but never used.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6225754)